wls-auth-service/src/main/resources/db/dummydata/oracle/R__setUserDataForWlsAllUsers.sql

```diff
@@ -1,21 +1,43 @@
--- update for user wls_all_bwb
+-- update for user wls_komw_bwb
 UPDATE Wlsuser
 SET wahlbezirkID     = '9081145c-dc16-4a9b-8255-5c49ddb98c14',
     email            = 'dummy@email.de',
```

Comment:
Um den User leichter zu finden, da der Username encrypted wird

```suggestion
    email            = 'wls_komw_bwb@email.de',
```